core/unit_test/TestOther.hpp

```diff
@@ -16,7 +16,6 @@
 
 #ifndef KOKKOS_TEST_OTHER_HPP
 #define KOKKOS_TEST_OTHER_HPP
-#include <TestAggregate.hpp>
```

Comment:
Should `TestAggregate.hpp` file be also removed?

Reply:
Sloppy rebase.  Fixed.  Thanks.